xgboost_synthetic/README.md

```diff
@@ -1,9 +1,11 @@
 # xgboost-synthetic
-Kubeflow fairing, pipelines demo using synthetic data
+Kubeflow fairing, pipelines demo using synthetic data. This notebook `build-train-deploy.ipynb` can be executed using one of the stock notebook images launched through Kubeflow UI.
 
-1. Launch a notebook
+1. Follow the [Set up your notebook](https://www.kubeflow.org/docs/notebooks/setup/) guide to get started with Jupyter notebooks on Kubeflow
 
-   ```
-   kubectl apply -f notebook.xgboost-synthetic.yaml
-   ```
-1. Attach an extra data volume named 
+1. Open a notebook and run
```

Comment:
Nit: is it better to open a terminal and run `git` command?

Reply:
Fixed. Thanks